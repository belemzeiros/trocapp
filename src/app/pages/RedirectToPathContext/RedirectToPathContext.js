const RedirectToPathContext = ({ staticContext = {} }) => {
  staticContext.url = process.env.PATH_CONTEXT;

  return null;
};

export default RedirectToPathContext;
