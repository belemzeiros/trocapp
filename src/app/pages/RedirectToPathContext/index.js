import RedirectToPathContext from './RedirectToPathContext';

export default RedirectToPathContext;
