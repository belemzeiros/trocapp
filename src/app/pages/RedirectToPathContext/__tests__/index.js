import React from 'react';
import { createShallow } from '@material-ui/core/test-utils';
import RedirectToPathContext from '../';

describe('RedirectToPathContext', () => {
  const shallow = createShallow();

  it('RedirectToPathContext will mount correctly', () => {
    const wrapper = shallow(<RedirectToPathContext />);
    expect(wrapper).toMatchSnapshot();
  });
});
