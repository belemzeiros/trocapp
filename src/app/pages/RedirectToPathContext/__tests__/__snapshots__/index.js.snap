// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RedirectToPathContext RedirectToPathContext will mount correctly 1`] = `""`;
